src/ru.ts

```diff
@@ -0,0 +1,2508 @@
+            earn: {
+                invest: "Посмотреть свои накопления",
+                withdraw: "Накопительный счёт",
+            },
+            home: "Домой",
+        },
+        amount: {
+            title: "Накопительный счёт",
+            estimate: "Примерный баланс: {{value}}",
+            position: "Подробности депозита",
+            continue: "Вклад",
+            available: "Доступно {{balance}} {{symbol}}",
+            insufficient: "Недостаточно средств",
+        },
+        work: {
+            title: "Как работает Накопительный счёт?",
+            subtext: "Clave - это кошелёк полностью самостоятельного управления, предоставляющий простой доступ к различным вкладам в децентрализованных финансовых сервисах, обеспечивая полное сохранение вашего контроля над средствами.",
+            continue: "Я готов попробовать",
+            defi: {
+                title: "Использование DeFi сервисов",
+                text: "Вам не нужно отдавать свои средства кому-либо для получения доходности.",
+            },
+            withdraw: {
+                title: "Гибкие выводы",
+                text: "Вы можете вывести вашу доходность и вклад в любое время.",
+            },
+            apr: {
+                title: "Динамичная ставка",
+                text: "Доходность берётся из комиссий торговой пары, к которой вы предоставляете ликвидность, и зависит от активности торговли на ней.",
+            },
+        },
+    },
+    hub: {
+        __description: "Hub Page and Quick actions in Dashboard",
+        title: "Центр",
+        all_actions: "Все функции",
+        start_with: "Начните с",
+        widget: {
+            title: "Быстрые действия",
+            see_all: "Открыть все",
+        },
+        actions: {
+            deposit: "Ввод средств",
+            deposit_text: "Пополните ваш кошелёк криптовалютой",
+            earn: "Накопительный счёт",
+            earn_text: "Получайте доходность за вашу криптовалюту в сети ZKsync Era",
+            links_text: "Создавайте и делитесь платёжными ссылками с друзьями.",
+            guardians: "Доверенные лица",
+            guardians_text: "Добавьте ваших друзей и близких как доверенных лиц",
+            browser: "Браузер",
+            links: "Платёжные ссылки",
+            split: "Разделённые платежи",
+            buy: "Приобрести",
+            invest: "Инвестировать",
+            get_started: "Начать",
+            benefits: "Ваши выгоды с Clave",
+            benefits_text: "Изучите ваши выгоды.",
+            soon: "(Скоро)",
+            portfolio: "Портфель активов",
+            onramp: {
+                title: "Банковский перевод",
+                detail: "Детали",
+                estimated_time: "Примерное время",
+                details: {
+                    accepting: "Перевод на сумму {{amount}} {{symbol}} в процессе.",
+                    purchasing: "USDC готовы к отправке.",
+                    sending: "Производится перевод USDC.",
```

Comment:
This is not important for Russian users but the onramped amount comes as USDC.e :(

```suggestion
                    accepting: "Перевод на сумму {{amount}} {{symbol}} в процессе.",
                    purchasing: "USDC.e готовы к отправке.",
                    sending: "Производится перевод USDC.e.",
```

Reply:
This is not just "not important". This is very confusing. Especially in the areas of extensive Tron usage (i.e. USDT), such as post-soviet, where people don't even know USDC, let aside USDT.

![image](https://github.com/user-attachments/assets/b8b7d0fe-5df7-40ef-8e9c-e4d181b400b4)

![image](https://github.com/user-attachments/assets/39d596b4-cb52-48e5-90dd-9622830964aa)

Reply:
FYI the only reason the second options are not at 0% is because I explained the difference when just onboarded them